testdata/p4_16_samples_outputs/psa-dpdk-hdr-field-non-align.p4-error

```diff
@@ -0,0 +1,4 @@
+psa-dpdk-hdr-field-non-align.p4(58): [--Wwarn=uninitialized_use] warning: flg may be uninitialized
+  if (flg == 0x2) {
```

Comment:
We can update the test to initialize this to avoid this warning. This isn't related to this issue. Maybe try making the flg as user metadata